roles/openshift_etcd_ca/tasks/main.yml

```diff
@@ -0,0 +1,9 @@
+---
+- include_role:
```

Comment:
Could we just move this inside of this role?  Almost feels like it belongs here or can we get rid of it entirely since it seems that it isn't doing much:
```
- openshift_facts:
    role: etcd
    local_facts:
      etcd_image: "{{ osm_etcd_image | default(None) }}"
```

If we move this inside of the role and just use the `osm_etcd_image` variable where it is called then I don't think we need to run `openshift_facts`.

Reply:
Atm, I am "almost" blindly moving tasks from one etcd role into another. Did not check what is actually inside and which variables are actually needed. I plan to do that once all the etcd roles are consolidated into one so I can open a PR with once change removing the etcd facts deps from all places at once wherever it is possible.

Checking the code I don't see any use of the `etcd_image` inside of the `etcd/tasks/ca/main.yml`.